templates/repo/issue/view_content.tmpl

```diff
@@ -39,7 +39,13 @@
 						<div class="ui bottom attached segment">
 							<div class="ui small images">
 								{{range .Issue.Attachments}}
-									<a target="_blank" href="{{AppSubUrl}}/attachments/{{.UUID}}"><img class="ui image" src="{{AppSubUrl}}/attachments/{{.UUID}}"></a>
+									<a target="_blank" href="{{AppSubUrl}}/attachments/{{.UUID}}">
+										{{if FilenameIsImage .Name}}
+											<img class="ui image" src="{{AppSubUrl}}/attachments/{{.UUID}}" title='{{printf ($.i18n.Tr "repo.issues.attachment.open_tab") .Name}}'>
```

Comment:
See the change on `conf/locale/locale_en-US.ini`. There's a `℅s` in this string, because the name of the file is being injected there.